.pnpmfile.cjs

```diff
@@ -275,6 +275,12 @@ function fixPeerDeps( pkg ) {
 		}
 	}
 
+	// Remove jQuery peer dependency, given it's already bundled in WordPress.
```

Comment:
Personally I'd (temporarily) install it as a dev dependency instead. But it doesn't matter a whole lot if the upgrade is going to happen soon.

Reply:
Yeah, I was torn. I'll leave it for now, but I anticipate it being irrelevant anyway.